api/src/test/java/bio/terra/tanagra/service/underlay/NauticalUnderlayUtils.java

```diff
@@ -35,6 +35,8 @@ public static bio.terra.tanagra.proto.underlay.Underlay loadNauticalUnderlayProt
   }
 
   public static final String NAUTICAL_UNDERLAY_NAME = "nautical_underlay";
+  public static final String NAUTICAL_CRITERIA_CONFIGS =
+      "[{\"type\":\"attribute\",\"title\":\"Age\",\"defaultName\":\"Age\",\"plugin\":{\"attribute\":\"age\"}]";
```

Comment:
See if you can use the `"""` string notation to avoid the escaped quotes.

Reply:
I tried using the `"""` notation but got the following complaint: `Text block literals are not supported at language level '11'`. Would escaped quotes be okay?